### DISCUSSION TITLE
Is it possible to add extra field to `mv-popup?

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#855] [Q&A] [answered]
Hi 

All demos I see to modify <a> element only modify the url of the link, but not the displayed text itself. 
(unless text is the same as href)

While looking for a solution to edit both properties, I saw it's possible to add 2 <meta> 
but I'm searching for some kind of more intuitive UI to modify both directly in edit mode in one step

I thought about using the mv-popup, with 2 fieldsets already included
1. url
2. text to display

Is it possible to add extra field to `mv-popup?`, so it would display 2 fieldsets?

Extra bonus, would be a dropdown to edit target(anchors inside page, or new page)

**Comment by LeaVerou** [comment, marked as answer]
Typically the way this is done is that you have a separate property (usually a `<span>`) for the text to display.
